frontend/src/components/utils/Update.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+interface Props {
+  category: string;
+  count: number;
+}
+export default ({ category, count }: Props) => {
+  if (count === 0) {
+    return null;
+  }
+  return (
+    <div className="update">
+      {count > 1 ? (
+        <p>
+          {count} new {category}s. Refresh or Click to view the latest{" "}
+          {category}s.
```

Comment:
I find string interpolation easier to maintain as it is a string:

```suggestion
          {`${count} new ${category}s. Refresh or Click to view the latest ${category}s.`}
```